README.md

```diff
@@ -1,101 +1,5 @@
-  Centre for Research on Environmental Systems and Statistics
-  Institute of Environmental and Biological Sciences
-  Lancaster University, Lancaster LA1 4YQ, UK
-
-  Tel: (+44) 1524 593892  Fax: (+44) 1524 593985
-  Email:  K.Beven@UK.AC.LANCASTER
+This model is in development for use in the [NGen Water Modeling Framework Prototype](https://github.com/NOAA-OWP/ngen). It includes wrapper functions for the [Basic Model Interface](https://bmi-spec.readthedocs.io/en/latest/).  More BMI related references can be found [here](refs/csdms).
   
-****************************************************************
-
-  SIMPLE SUBCATCHMENT VERSION OF TOPMODEL
-
-  This program allows single or multiple subcatchment calculations 
-  but with single average rainfall and potential evapotranspiration
-  inputs to the whole catchment.  Subcatchment discharges are routed
-  to the catchment outlet using a linear routing algorithm with
-  constant main channel velocity and internal subcatchment 
-  routing velocity.  The program requires ln(a/tanB) distributions
-  for each subcatchment.  These may be calculated using the
-  GRIDATB program which requires raster elevation data as input.
-  It is recommended that those data should be 50 m resolution or
-  better.
-
-  NOTE that TOPMODEL is not intended to be a traditional model
-  package but is more a collection of concepts that can be used
-  **** where appropriate ****. It is up to the user to verify that
-  the assumptions are appropriate (see discussion in 
-  Beven et al.(1994).   This version of the model  will be
-  best suited to catchments with shallow soils and moderate
-  topography which do not suffer from excessively long dry 
-  periods.  Ideally predicted contributing areas should be
-  checked against what actually happens in the catchment.
-
-  It includes infiltration excess calculations and parameters
-  based on the exponential conductivity Green-Ampt model of 
-  Beven (HSJ, 1984) but if infiltration excess does occur it
-  does so over whole area of a subcatchment.  Spatial variability
-  in conductivities can however be handled by specifying 
-  Ko parameter values for different subcatchments, even if they
-  have the same ln(a/tanB) and routing parameters, ie. to 
-  represent different parts of the area. 
-
-  Note that time step calculations are explicit ie. SBAR
-  at start of time step is used to determine contributing area.  
-  Thus with long (daily) time steps contributing area depends on 
-  initial value together with any volume filling effect of daily 
-  inputs.  Also baseflow at start of time step is used to update 
-  SBAR at end of time step
-
-  Current program limits are:
-          Number of time steps = 2500
-          Number of subcatchments = 10
-          Number of ln(a/tanB) increments = 30
-          Number of subcatchment routing ordinates = 10
-          Number of time delay histogram ordinates = 20
-          Size of subcatchment pixel maps = 100 x 100
-
```

Comment:
There are this item and a few other similar things being removed from this file that seem like they could still be important (in addition to the usage conditions, which I commented on above).  Is this all either no longer applicable or documented elsewhere?

Reply:
Again, still stated in [topmod.c](https://github.com/NOAA-OWP/topmodel/blob/b05a89430525161a5d086fdfb480e434c417f428/src/topmodel.c#L73). 
fyi - don't believe max Number of time steps = 2500 is ever enforced; will confirm & comment accordingly.  Other limits here, however, are set in bmi initialize()